src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/CO2Solubility.cpp

```diff
@@ -248,7 +248,12 @@ TableFunction const * makeSolubilityTable( string_array const & inputParams,
   string const tableName = functionName + "_co2Dissolution_table";
```

Comment:
I think we should allow the user to give the table a name instead of these internally generated names. One use case would be where I want to use the same table across say 3 fluids.

Reply:
just not quite sure how to pull down the table name here without changing too much :)

Reply:
@paveltomin I put a comment on the associated issue #2810. As for how to implement it, let me draft a change.